rs/nns/governance/src/governance.rs

```diff
@@ -2391,22 +2403,20 @@ impl Governance {
             return Err(GovernanceError::new(ErrorType::NotAuthorized));
         }
 
-        if split_amount_e8s < min_stake + transaction_fee_e8s {
+        let min_split_amount = min_stake.saturating_add(transaction_fee_e8s);
```

Comment:
I like that you replace split_amount_e8s with split.amount_e8s. To make this even better, maybe change line 2377 to

```
amount_e8s: _,
```

so that split_ammount_e8s is not a pollution.